students/anthony_mckeever/lesson2/exercise_3/series.py

```diff
@@ -0,0 +1,85 @@
+"""
+Programming In Python - Lesson 2 Exercise 3: Fibonacci Series Exercise
+Code Poet: Anthony McKeever
+Start Date: 07/24/2019
+End Date: 07/24/2019
+"""
+
+def fibonacci(n):
+    """
+    Return the Fibonacci number at the nth place.
+
+    :param n:   The number of the place in the Fibonacci series to get.  For example, n = 7 will return 13 
+    """
+    if n == 0 or n == 1:
+        return n
+    return fibonacci(n - 2) + fibonacci(n - 1)
+
+def lucas(n):
+    """
+    Return the Lucas number at the nth place
+
+    :param n:   The number of the place in the Lucas series to get.  For example, n = 7 will return 29
+    """
+    if n == 0:
+        return 2
+    if n == 1:
+        return 1
+
+    return lucas(n - 1) + lucas(n - 2)
+
+def sum_series(n, atZero = 0, atOne = 1):
```

Comment:
Make these camel_case too (almost everything is camel_case in Python except Class names :) . Also, no whitespace around the equal signs.